test/integration/exceptionalOpeningTimes.js

```diff
@@ -18,38 +18,55 @@ function mockCurrentDate2() {
 }
 
 function expectExceptionalOpeningTimes1($, rows, times) {
-  expect($(rows[0]).text()).to.include('Monday 29 May');
+  expect($(rows[0]).text()).to.include('Thursday 25 May');
   expect($(rows[0]).text()).to.include(times[0]);
+  expect($(rows[1]).text()).to.include('Monday 29 May');
+  expect($(rows[1]).text()).to.include(times[1]);
 }
 
```

Comment:
This function is only used once, rather than passing the array of times in it would be clearer to replace times[0] and times[1] with the value in the function and remove the times parameter.